lib/feature/pokemon_details_page.dart/pokemon_details_page.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:pokedex_async_redux/api/models/pokemon_model.dart';
+
+import 'package:flutter/material.dart';
+
+class PokemonDetails extends StatelessWidget {
+  const PokemonDetails({this.pokemon});
+  final Pokemon? pokemon;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        leading: BackButton(
+          color: Colors.black,
+        ),
+        title: const Text('Pokédex', style: TextStyle(color: Colors.black)),
```

Comment:
trailing comma, since we have an unnamed parameter and a named parameter in the Text Widget